wman-backend/Wman.Test/Tests/EventLogicTests.cs

```diff
@@ -23,22 +25,102 @@ public class EventLogicTests
         private List<WorkEvent> eventList;
         private List<AddressHUN> addressList;
 
+        private Mock<UserManager<WmanUser>> userManager;
+        private List<WmanUser> users;
+
         [SetUp]
         public void SetUp()
         {
+            this.users = UserManagerBuilder.GetWmanUsers();
+            this.userManager = UserManagerBuilder.GetUserManager(users);
+
+            this.mapper = MapperBuilder.GetMapper();
+
             this.eventList = EventLogicBuilder.GetWorkEvents();
             this.addressList = EventLogicBuilder.GetAddresses();
-            this.mapper = EventLogicBuilder.GetMapper();
 
             this.eventRepo = EventLogicBuilder.GetEventRepo(eventList);
             this.addressRepo = EventLogicBuilder.GetAddressRepo(addressList);
         }
 
+        [Test]
+        public async Task MassAssignUser_AssignMultipleUsers_Successfull()
+        {
+            //Arrange
+            EventLogic eventLogic = new EventLogic(this.eventRepo.Object, this.mapper, this.addressRepo.Object, this.userManager.Object);
+            List<string> userNames = new List<string>() { users[0].UserName, users[1].UserName };
+
+            //Act
+            var call = eventLogic.MassAssignUser(eventList[0].Id, userNames);
+
+            //Assert
+            Assert.That(call.IsCompleted);
+
+            this.eventRepo.Verify(x => x.GetOneWithTracking(It.IsAny<int>()), Times.Once);
+            this.userManager.Verify(x => x.Users, Times.Exactly(2));
+            this.eventRepo.Verify(x => x.Update(It.IsAny<int>(), It.IsAny<WorkEvent>()), Times.Exactly(2));
```

Comment:
```suggestion
            this.eventRepo.Verify(x => x.Update(It.IsAny<int>(), It.IsAny<WorkEvent>()), Times.Exactly(userNames.Count));
```
The same logic applies here as above.

Reply:
you got it boss 👌

Reply:
Perfect 👏